### ISSUE TITLE
Build failure in 0.22.0-alpha1 on Linux

### ISSUE THREAD

**Issue by kvark** [rust-windowing/glutin#1204]
See https://travis-ci.org/gfx-rs/gfx/jobs/572262934#L794

```
    Checking glutin v0.22.0-alpha1

error[E0599]: no method named `is_wayland` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/mod.rs:100:15

    |

100 |         if el.is_wayland() {

    |               ^^^^^^^^^^

    |

    = help: items from traits can only be u    Checking glutin v0.22.0-alpha1

error[E0599]: no method named `is_wayland` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/mod.rs:100:15

    |

100 |         if el.is_wayland() {

    |               ^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `is_wayland` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/mod.rs:136:15

    |

136 |         if el.is_wayland() {

    |               ^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `wayland_display` found for type `&winit::event_loop::EventLoop<T>` in the current scope

  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/wayland.rs:55:30

   |

55 |         let display_ptr = el.wayland_display().unwrap() as *const _;

   |                              ^^^^^^^^^^^^^^^

   |

   = help: items from traits can only be used if the trait is in scope

   = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

           `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `xlib_xconnection` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/x11.rs:212:30

    |

212 |         let xconn = match el.xlib_xconnection() {

    |                              ^^^^^^^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `xlib_xconnection` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/x11.rs:469:30

    |

469 |         let xconn = match el.xlib_xconnection() {

    |                              ^^^^^^^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

sed if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `is_wayland` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/mod.rs:136:15

    |

136 |         if el.is_wayland() {

    |               ^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `wayland_display` found for type `&winit::event_loop::EventLoop<T>` in the current scope

  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/wayland.rs:55:30

   |

55 |         let display_ptr = el.wayland_display().unwrap() as *const _;

   |                              ^^^^^^^^^^^^^^^

   |

   = help: items from traits can only be used if the trait is in scope

   = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

           `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `xlib_xconnection` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/x11.rs:212:30

    |

212 |         let xconn = match el.xlib_xconnection() {

    |                              ^^^^^^^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

error[E0599]: no method named `xlib_xconnection` found for type `&winit::event_loop::EventLoop<T>` in the current scope

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha1/src/platform_impl/unix/x11.rs:469:30

    |

469 |         let xconn = match el.xlib_xconnection() {

    |                              ^^^^^^^^^^^^^^^^

    |

    = help: items from traits can only be used if the trait is in scope

    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:

            `use winit::platform::unix::EventLoopWindowTargetExtUnix;`

```

**Comment by ZeGentzy** [comment]
`cargo update`. Get it to use the latest winit and glutin and this will go away.

**Comment by XeroOl** [comment]
I get this same issue with alpha-2.

**Comment by ZeGentzy** [comment]
I had some fixes lying around that I'd forgotten to push a while back. Should be fixed in alpha3, which will release when #1206 is merged.

**Comment by kvark** [comment]
Thanks! This is resolved for us now in alpha3.
